### PR TITLE
Fixed envs

### PR DESCRIPTION
**Pull request by martin-s-a**
- Modeller now inherits from pwchem
- Fixed env calls

### CODE REVIEW

File: modellerScipion/__init__.py [hunk 1]
```diff
@@ -27,35 +27,41 @@
...
-class Plugin(pwem.Plugin):
+class Plugin(pwchem.Plugin):
     _supportedVersions = [MODELLER_DIC['version']]
 
     @classmethod
     def _defineVariables(cls):
-        """ Return and write a variable in the config file.
-        """
+        """ Return and write a variable in the config file. """
         cls._defineVar("MODELLER_ENV_ACTIVATION", 'conda activate modeller-env')
```
Comment (DaniDelHoyo) [inline modellerScipion/__init__.py]: shouldn't we change this to the new env name? or if it is not used with the new conf , delete it

Comment (DaniDelHoyo) [inline modellerScipion/__init__.py]: shouldn't we change this to the new env name? or if it is not used with the new conf , delete it

Comment (martin-s-a) [reply modellerScipion/__init__.py]: Changed with home var, not used for now, but will be when new installation method is implemented

File: modellerScipion/__init__.py [hunk 2]
```diff
@@ -27,35 +27,41 @@
...
+    def addModellerPackage(cls, env):
+        """ This function installs Modeller package. """
         MODELLER_INSTALLED = 'modeller_installed'
 
         installationCmd = cls.getCondaActivationCmd()
-        installationCmd += ' conda create -y -c salilab -n modeller-env modeller={} &&'.format(MODELLER_DIC['version'])
-        installationCmd += ' touch %s' % MODELLER_INSTALLED
+        installationCmd += ' conda create -y -c salilab -n {} modeller={} &&'.format(cls.getEnvName(MODELLER_DIC), MODELLER_DIC['version'])
+        installationCmd += ' touch %s &&' % MODELLER_INSTALLED
+        installationCmd += ' {} && export MODELLER_CONDA_PREFIX=$CONDA_PREFIX &&'.format(cls.getEnvActivationCommand(MODELLER_DIC, condaHook=False))
```
Comment (DaniDelHoyo) [inline modellerScipion/__init__.py]: i am not convinced about this env variable for storing the env path. also, as it is, seems like the env path is not stored prior to its first use here, so the first time you install the plugin, it won't be printed

**Review by DaniDelHoyo — CHANGES_REQUESTED** [review]

File: modellerScipion/__init__.py (hunk 2 above)
Comment (martin-s-a) [reply modellerScipion/__init__.py]: Yes, i am wroking in a full modification of the installation process like in scipion-chem and scipion-chem-autodock. As soon as it is ready, i'll push it to this branch, and this variable won't be used any more

**Review by DaniDelHoyo — APPROVED** [review]

**Pull request merged**